opentreemap/treemap/migrations/0034_add_permission_view_external_link.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+# Note: to reuse this data migration when adding a new permission
+# in the future, copy the code and update these two lines:
+
+_new_permission_codename = 'view_external_link_on_detail_page'
```

Comment:
Good point. I added that because the name "external link" seems too general for this feature -- there could be all kinds of "external link". But this is the one we've got... changed to `view_external_link`.